core/arch/arm/kernel/thread.c

```diff
@@ -1163,7 +1163,7 @@ DECLARE_KEEP_PAGER(thread_system_reset_handler);
 #endif /*CFG_WITH_ARM_TRUSTED_FW*/
 
 #ifdef CFG_CORE_WORKAROUND_ARM_NMFI
-void __noreturn itr_core_handler(void)
+void __noreturn interrupt_main_handler(void)
```

Comment:
unrelated change.
Please rebase your commit on github.com/OP-TEE/optee_os.git master branch tip.

Reply:
I added it to compile our code, meanwhile I reverted it to original